firedrills.md

```diff
@@ -168,6 +192,40 @@ node-1-planet$ journalctl -u kube-apiserver --no-pager
 node-1-planet$ journalctl -u kube-kubelet --no-pager
 ```
 
+Most of these services run with standard logging settings (usually logging at INFO level) and you may want to increase the verbosity of the log output of a certain component when troubleshooting something. Different processes have different ways of increasing the log verbosity, Kubernetes services usually take a `-v` flag followed by a numeric verbosity level, 1-10.
+
+Let's see how we can get kubelet to log more information.
+
+First, we need to identify where its systemd unit file is located as we'll need to edit it to add the flag. Depending on the service, its unit file may be found in `/lib/systemd/system` or `/etc/systemd/system` so to know for sure let's check the unit status:
+
+```bash
+node-1-planet$ systemctl status kube-kubelet
+● kube-kubelet.service - Kubernetes Kubelet
+   Loaded: loaded (/lib/systemd/system/kube-kubelet.service; static; vendor preset: enabled)
+   Active: active (running) since Mon 2019-10-07 19:19:13 UTC; 3 months 9 days ago
+```
+
+Once the unit's service spec file is identified, edit it and add a `-v5` flag somewhere to the `ExecStart` command:
+
+```bash
+node-1-planet$ nano /lib/systemd/system/kube-kubelet.service
+```
+
+Once the file has been edited, we need to tell systemd to reload the configuration, otherwise the changes won't take effect:
+
+```bash
+node-1-planet$ systemctl daemon-reload
+```
+
+Finally, we can restart kubelet which will start producing more debug output in its logs:
+
+```bash
+node-1-planet$ systemctl restart kube-kubelet
+node-1-planet$ journalctl -u kube-kubelet -f
+```
+
+Note that the changes we've made to the systemd unit will persist across the Planet container restarts but will be reset when a new version of Planet is rolled during system upgrade, for example.
```

Comment:
> Note that the changes we've made to the systemd unit will persist across the Planet container restarts but will be reset when a new version of Planet is rolled during system upgrade, for example.

Maybe something more along the lines of:

Note that the changes we've made to the systemd unit will persist across Planet container restarts but will be lost during a gravity upgrade that includes a new version of planet due to the inclusion of a new rootfs.